public/about.html

```diff
@@ -27,39 +27,42 @@
    <!-- End Common Head -->
    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ==" crossorigin>
    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js" integrity="sha512-/Nsx9X4HebavoBvEBuyp3I7od5tA0UzAxs+j83KgC8PU0kgB4XiK4Lfe4y4cgBtaRJQEIFCW+oC506aPT2L1zw==" crossorigin></script>
-</head>
 
-<body>
+
+
    <header>
+   <a id="skip-nav" class="sr-only sr-only-focusable" href="#main-content">
+      Skip Navigation or Skip to Content
+    </a>
       <nav class="navbar navbar-expand-lg navbar-light fixed-top bg-white py-1 px-2">
          <div class="container">
             <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
             </button>
-            <a class="navbar-brand" title="BLE Home Page" href="index.html"><img src="img/logo-round-rgb.png" style="max-height: 40px;" alt="BLE home
+            <a class="navbar-brand" title="BLE Home Page" href="index.html" role="button"><img src="img/logo-round-rgb.png" style="max-height: 40px;" alt="BLE home
```

Comment:
Why is role="button" needed here?

Reply:
here's the relevant bit from the checker:
![image](https://user-images.githubusercontent.com/42045942/116714503-756c7800-a99b-11eb-8fcf-6c752f53700a.png)

Reply:
Which elements did we get dinged on?  The nav items that truly act as links are fine without the role. The nav links that are just drop down toggles should have role="button".

Reply:
yup, you're correct. Just the drop down toggles, so about, research, and data

Reply:
So for line 42 here, role should be removed.

Reply:
done at 3c178a14f71b631a502d614ab2edf0b4e02e963e